app/src/sharedTest/java/org/oppia/android/app/utility/ProtoExtraMatcher.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.app.utility
+
+import android.content.Intent
+import com.google.protobuf.MessageLite
+import org.hamcrest.Description
+import org.hamcrest.Matcher
+import org.hamcrest.TypeSafeMatcher
+import org.oppia.android.util.extensions.getProtoExtra
+
+/** A custom matcher to check if an Intent has a specific proto extra. */
+class ProtoExtraMatcher {
+  companion object {
+    fun <T : MessageLite> hasProtoExtraCheck(keyName: String, expectedProto: T):
```

Comment:
Please add a kdoc to this function and remove the kdoc exemption.

Reply:
Done.

Reply:
@Vishwajith-Shettigar, you seem to have missed removing the file from exemptions. See static check failure.

Reply:
ahh!, right. I'll remove, thank you.